docs/UserGuide.md

```diff
@@ -37,11 +43,72 @@
 
 ## Features
 
+<box type="info" seamless>
+
+**Notes about the command format:** <br />
+
+* Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
+e.g. in `add n/NAME`, `NAME` is a parameter which can be used as `add n/John Doe`.
+
+* Items in square brackets are optional.<br>
+e.g `n/NAME [t/TAG]` can be used as `n/John Doe t/friend` or as `n/John Doe`.
+
+* Items with `…`​ after them can be used multiple times including zero times.<br>
+e.g. `[t/TAG]…​` can be used as ` ` (i.e. 0 times), `t/friend`, `t/friend t/family` etc.
+
+* Parameters can be in any order.<br>
+e.g. if the command specifies `n/NAME p/PHONE_NUMBER`, `p/PHONE_NUMBER n/NAME` is also acceptable.
+
+* Extraneous parameters for commands that do not take in parameters (such as `help`, `list`, `exit` and `clear`) will be ignored.<br>
+e.g. if the command specifies `help 123`, it will be interpreted as `help`.
+
+* If you are using a PDF version of this document, be careful when copying and pasting commands that span multiple lines as space characters surrounding line-breaks may be omitted when copied over to the application.
+</box>
+
+--------------------------------------------------------------------------------------------------------------------
+### Viewing all guests: `guest list`
+View all guests in a list format.
+
+```text
+guest list
+```
+
+Expected behaviour upon success:
+- Displays a list of all guest names and their respective indexes. (Example: 1. Marcus Tan, 2. Jane Lim)
+
+Expected behaviour upon failure:
+<br>(refer to Appendix A : Expected behaviour upon general failure)
+
+--------------------------------------------------------------------------------------------------------------------
+### View a specific guest: `guest view`
+View a specific guest using a specified index.
+
+```text
+guest view INDEX
+```
+
+Acceptable values for INDEX
+- A positive integer
+
+Examples:
+`guest view 1`
+
+Expected behaviour upon success:
+- Displays a guest and all the information associated with it. (Example: 1. Marcus Tan)
+
+Expected behaviour upon failure:
+- Number out of index range, not a number, or no number: Displays error message “Please input a positive integer as the index.”
+- Number does not correspond to any guest: Displays error message “The number you have provided does not correspond to any guest.”
+- No input number : Displays error message “Please input an index”
```

Comment:
I noticed that some colons have a space before and some don't. Perhaps this could be standardized? I noticed the same issue in a few places.